be/src/vec/functions/function_json.cpp

```diff
@@ -833,7 +833,7 @@ struct FunctionJsonQuoteImpl {
 };
 
 struct FunctionJsonExtractImpl {
-    static constexpr auto name = "json_extract";
+    static constexpr auto name = "json_extract_old";
```

Comment:
If the logic of function `json_extract()` is moved to `function_jsonb.cpp`, is it alright to delete this `FunctionJsonExtractImpl` further more?

Reply:
I hope so, but regress test has some cases fail so far , maybe it can not moved....